PoC-differ/content.go

```diff
@@ -31,10 +31,6 @@ const (
 	directoryAttribute = "directory"
 )
 
-func (err types.MissingMandatoryFile) Error() string {
```

Comment:
moved to types.go as we cannot define a method on a type outside of its package.